client/core/core.go

```diff
@@ -1102,9 +1166,14 @@ out:
 				case msgjson.BookOrderRoute:
 					err = c.handleBookOrderMsg(dc, msg)
 				case msgjson.EpochOrderRoute:
-					log.Info("epoch_order message received")
+					err = c.handleEpochOrderMsg(dc, msg)
 				case msgjson.UnbookOrderRoute:
 					err = c.handleUnbookOrderMsg(dc, msg)
+				case msgjson.MatchProofRoute:
+					// TODO: since handling match proof messages is optional
+					// there should be an associated config option to toggle it
+					// right?
```

Comment:
Let's just have the clients do it for now.

Reply:
If we're receiving the match proof, we might as well check it. The real optimization will come when the work for #142 is done. The "lite subscription" won't receive these messages.